alb-lambda-rest-api-sam-py/README.md

```diff
@@ -0,0 +1,154 @@
+# AWS Lambda RESTful API with Amazon ALB and Path Based Listener Rules
+
+This configuration pattern creates an Application Load
+Balancer with path-based listener rules, paired with an AWS Lambda RESTful API function as the target. It uses  Python 3.9 and the Serverless Application Model (SAM) CLI.
+
+Learn more about this pattern
+at [Serverless Land Patterns](https://serverlessland.com/patterns/alb-lambda-rest-api-sam-py).
+
+Important: this application uses various AWS services and there are costs associated with these services after the Free
+Tier usage - please see the [AWS Pricing page](https://aws.amazon.com/pricing/) for details. You are responsible for any
+AWS costs incurred. No warranty is implied in this example.
+
+## Requirements
+
+* [Create an AWS account](https://portal.aws.amazon.com/gp/aws/developer/registration/index.html) if you do not already
+  have one and log in. The IAM user that you use must have sufficient permissions to make necessary AWS service calls
+  and manage AWS resources.
+* [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/install-cliv2.html) installed and configured
+* [Git Installed](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
+* [AWS Serverless Application Model](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html) (
+  AWS SAM) installed
+* [Python 3 installed](https://www.python.org/downloads/)
+* [AWS Lambda Powertools for Python](https://docs.powertools.aws.dev/lambda/python/latest/)
+* [Docker](https://www.docker.com/products/docker-desktop/)
+
+## Deployment Instructions
+
+1. Create a new directory, navigate to that directory in a terminal and clone the GitHub repository:
+    ``` 
+    git clone https://github.com/aws-samples/serverless-patterns
+    ```
+2. Change directory to the pattern directory:
+    ```
+    cd alb-lambda-rest-api-sam-py
+    ```
+3. Install dependencies and build:
+    ```
+    pipenv install
```

Comment:
If I do not run `pipenv requirements > requirements.txt` before `sam deploy` the Powertools dependency is not installed. Is the command missing here or did I miss something?

Reply:
I had this step in Testing the example locally. But it makes sense to have the instruction here as well.